src/engine/ruleset/decoders/f5/f5-apm-own.yml

```diff
@@ -0,0 +1,451 @@
+---
+name: decoder/f5-apm-own/0
+
+metadata:
+  module: F5
+  title: F5 APM Decoder for own providers
+  description: Decodes HEADER F5 BIG-IP Access Policy Manager logs of own providers.
+  compatibility: >
+    New Wazuh Engine
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/10
+  references:
+    - TBD
```

Comment:
Improve (Documentation): If is possible, add reference to oficial documentation

Reply:
agree